apps/aesophia/src/aeso_ast_to_icode.erl

```diff
@@ -235,6 +237,15 @@ ast_body({record,Attrs,{typed,_,Record,RecType={record_t,Fields}},Update}) ->
 ast_body({typed, _, Body, _}) ->
     ast_body(Body).
 
+%% Implemented as a contract call to the contract with address 0.
+prim_call(Prim, Amount, Args, ArgTypes, OutType) ->
+    #prim_call_contract{ gas      = #integer{ value = 0 },
+                         address  = #integer{ value = 0 },
```

Comment:
I see address 0 mentioned in (technical) documentation in module [`aevm_ae_primops`](https://github.com/aeternity/epoch/pull/1183/files#diff-a14025d96267bdad00e5afcaf65f98c8R5).

Is it address in ROM or memory or storage or (unlikely) contract tree under consensus?

It seems this address 0 not documented in protocol repo? Maybe shall fit [here](https://github.com/aeternity/protocol/blob/cde08c291750614a5908973100657ccbd69bf92f/contracts/sophia.md#contract-primitives)?

Maybe the hardcoded addresses shall be preferred in the code as macros or functions (rather than hardcoded values)?

Reply:
There is a ticket for documenting this https://www.pivotaltracker.com/story/show/158023467